app/models/concerns/user_unread_notifications_concern.rb

```diff
@@ -0,0 +1,6 @@
+module UserUnreadNotificationsConcern
```

Comment:
UserNotificationConcern, we can also add other notification related functions here.